### ISSUE TITLE
Add option to run CheckQC after demultiplexing with bcl2fastq

### ISSUE THREAD

**Issue by matrulda** [nf-core/demultiplex#143]
### Description of feature

Hi! 

We have an application to check if our sequencing runs fulfill certain (customisable) QC criteria after demultiplexing: https://github.com/Molmed/checkQC. We are considering using the `demultiplex` pipeline to do our demultiplexing in the future, and I'm wondering if it would be in scope to include an option to run CheckQC as part of the pipeline. Right now it only supports bcl2fastq output, but will support BclConvert in the future. Me and my colleagues are willing to work on this feature. 

Let me know what you think :)

**Comment by Emiller88** [comment]
That sounds awesome! Supports more people, worse case if someone doesn't want to run `checkQC` then they just skip it? Am I missing a drawback?

Would you or your colleges be interested in creating a nf-core module and then creating the PR to add it to this pipeline?

**Comment by matrulda** [comment]
Yeah, exactly! Totally skippable. I don't see any drawbacks with it, my only concern was if you would find it to be out of scope of what shold be included in this pipeline. 

We can definitely look into making a nf-core module for checkQC :+1:

**Comment by apeltzer** [comment]
We will work on this being added to demultiplexing, stay tuned. Have it running internally already, will contribute that now upstream here in demultiplexing.
